src/player/mod.rs

```diff
@@ -74,33 +74,31 @@ fn spawn_player(
             },
             pbr: PbrBundle {
                 mesh: meshes.add(Mesh::from(shape::Cube {
-                    size: PLAYER_SIZE as f32,
+                    size: PLAYER_SIZE,
                 })),
                 material: materials.add(Color::rgb(0.8, 0.7, 0.6).into()),
-                transform: Transform::from_xyz(0., PLAYER_SIZE as f32 / 2., 0.),
+                transform: Transform::from_xyz(0., PLAYER_SIZE / 2., 0.),
                 ..Default::default()
             },
         })
-        .insert(Player)
-        .insert(Health(PLAYER_HEALTH_MAX))
-        .insert(Velocity {
-            ..Default::default()
-        })
-        .insert(RotationConstraints::lock())
-        .insert(PhysicMaterial {
-            restitution: 0.,
-            friction: 0.0,
-            density: 1000.0,
-        })
-        .insert(RigidBody::Dynamic)
-        .insert(CollisionShape::Cuboid {
-            half_extends: Vec3::splat(PLAYER_COLLISION_SIZE),
-            border_radius: None,
-        })
-        .insert(
+        .insert_bundle((
```

Comment:
In bevy, it's preferable to use a single `insert_bundle` over many `insert` when possible, tuples let you do just that.